pkgs/applications/graphics/tesseract/wrapper.nix

```diff
@@ -18,7 +18,9 @@ let
 
     nativeBuildInputs = [ makeWrapper ];
 
-    buildCommand = ''
+    phases = [ "buildPhase" "fixupPhase" ];
```

Comment:
phases should not be set in packages

Reply:
This PR still definitely seemed like an improvement, though.

Reply:
#368545